petfinder-data/data_getter.py

```diff
@@ -165,7 +164,7 @@ def get_animals(
         "type": type,
         "status": status,
         "organization": organization,
-        "location": "Chicago, IL",
+        "location": City_State,
```

Comment:
Same option here -- since `City_State` is a variable, we could add it to the function parameters.

Reply:
Added